Cargo.toml

```diff
@@ -18,11 +18,11 @@ axum = "0.8.1"
 base64 = "0.22.1"
 clap = { version = "4.5.30", features = ["derive", "env"] }
 futures = "0.3"
-holochain_client = "0.6.2"
-holochain_conductor_api = "0.4.1"
-holochain_types = "0.4.1"
-holochain_websocket = "0.4.1"
-serde = { version = "=1.0.203", features = ["derive"] }
+holochain_client = "0.7.1"
+holochain_conductor_api = "0.5.3"
+holochain_types = "0.5.3"
+holochain_websocket = "0.5.3"
+serde = { version = "=1.0.219", features = ["derive"] }
```

Comment:
I think we don't have to pin serde if we depend on `holochain_serialized_bytes` because that pins it for us. It doesn't hurt but could maybe save some future typing by setting this to "1"?